### ISSUE TITLE
Add Korean, Indonesian, and Hebrew support

### ISSUE THREAD

**Issue by dlawrie** [hltcoe/patapsco#47]
Support the above languages in Patapsco

**Comment by cash** [comment]
Stop words were merged in from pull request #48. I tested the pipeline on some Korean documents a few months back. I think Patapsco defaulted to the UD tokenization model. I had someone who reads Korean take a look and she thought it was reasonable. UD tokenization stats here: https://explosion.ai/blog/ud-benchmarks-v3-2

So short answer is that you can set the language code and Patapsco should just work for Korean.

I will note that there is an issue when running Patapsco with multiple processes the first time it tries to download a model - basically a race condition - I need to restructure how the models get downloaded automatically in the multiprocessing setting.

**Comment by isoboroff** [comment]
Indeed, with an up-to-date repo Korean indexes fine and passes sanity-check searches.

NTCIR 3-6 has Korean data in their CLIR track.  I've reached out to Noriko Kando to get the data, but I bet Paul McNamee has it already.
